docs/sync-storage-handbook/src/README.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+Application Services are a set of libraries for connecting Mozilla products to the Firefox ecosystem. As we make Firefox available on more platforms, and grow from Firefox with entirely new products, we've found some common needs: signing in to a Firefox Account, syncing data through Firefox Sync, and receiving push messages. Application Services offers integrations for these services, so that products don't have to write their own.
+
+- What does "connected ecosystem" mean?
+- Why is storage integrated with Sync?
+- Should we talk about push and FxA elsewhere?
```

Comment:
Yes, I think we should, perhaps as sibling books in some sort of "connected applications trilogy" (or just a non-metaphor-stretching nested documentation structure)